src/test/java/com/ericsson/gerrit/plugins/eiffel/listeners/TestAbstractEventListener.java

```diff
@@ -61,7 +61,7 @@ public void testVerifyPluginEnabledForMyBranch() throws Throwable {
     public void testVerifyPluginEnabledForMyBranchUsingMultipleFilter() throws Throwable {
         boolean enabled;
         when(pluginConfig.isEnabled()).thenReturn(true);
-        when(pluginConfig.getFilter()).thenReturn("nope nupe my-branch nepp nupp");
+        when(pluginConfig.getFilter()).thenReturn("nope,nupe,my-branch,nepp,nupp");
```

Comment:
In the Javadock you state that it is possible to write ,<space>. so test that it works using "my-branch, nepp". I would want both ",", ", " and ",      " to work. Maybe it will work right away or the regexString might need to be trimmed.

Reply:
fixed in https://github.com/eiffel-community/eiffel-gerrit-plugin/pull/35/commits/e36e7b6785883f77ba3a31cf533b606c8095e9cc

Added a test case to see it works´.